web/packages/test-helpers/src/generateBeefyCheckpointProd.ts

```diff
@@ -0,0 +1,58 @@
+import { ApiPromise, WsProvider } from "@polkadot/api"
+import { MerkleTree } from "merkletreejs"
+import createKeccakHash from "keccak"
+import { publicKeyConvert } from "secp256k1"
+import type {
+    ValidatorSetId,
+    BeefyId,
+} from "@polkadot/types/interfaces/beefy/types"
+import fs from "fs"
+import path from "path"
+import { u32, u64 } from "@polkadot/types-codec";
+import { H256 } from "@polkadot/types/interfaces";
+import { Struct } from "@polkadot/types";
+
+interface AuthoritySet extends Struct {
+    id: u64;
+    len: u32;
+    keysetCommitment: H256;
+}
+
+async function generateBeefyCheckpoint() {
+    let beefyBlock = parseInt(process.env.BEEFY_BLOCK!, 10)
+    if (!Number.isInteger(beefyBlock) || beefyBlock == 0) {
+        throw new Error("InvalidBeefyBlock")
+    }
+
+    let _api = await ApiPromise.create({
+        provider: new WsProvider("wss://polkadot-rpc.dwellir.com"),
+    })
+
+    let api = await _api.at(await _api.rpc.chain.getBlockHash(beefyBlock))
+
+    let authorities = await api.query.beefyMmrLeaf.beefyAuthorities<AuthoritySet>()
+    let nextAuthorities = await api.query.beefyMmrLeaf.beefyNextAuthorities<AuthoritySet>()
```

Comment:
Assuming we reuse it for local setup, 
https://github.com/Snowfork/snowbridge/blob/238a8fcf304dab8d824c0c62adb07a2786df5012/web/packages/test-helpers/src/generateBeefyCheckpoint.ts#L72-L78 FYI.